### PR TITLE
replace RMSProp with adam as optimizer

### PR DESCRIPTION
**Pull request by McLawrence**
Adam with default parameters converges faster here than the RMSProp (At least during the first 50 epochs)

### CODE REVIEW

**Comment by McLawrence** [comment]
I don't see, how the `tests/keras/test_callbacks.py::test_TensorBoard_convnet ` test could be affected by the change

**Comment by Dref360** [comment]
CLose and reopen your PR to rerun the tests

**Comment by fchollet** [comment]
But why? You haven't provided any concrete justification for this change.

**Comment by McLawrence** [comment]
Due to hardware limitations, I first only compared the training and validation loss during the first 25 epochs. 
Here adam constantly achieved lower loss than RMSProp. 
However, I do not achieve a logloss of 0.55 after 50 epochs (as described in the comment in the code) with any of the optimizers. For both optimizers it seems to me that the training loss converges to approx. 0.75 (even after 100 epochs) but adam reaches this threshold faster.
